venter/admin.py

```diff
@@ -42,40 +42,41 @@ class AuthoritiesModelAdmin(admin.ModelAdmin):
 
 class ComplaintModelAdmin(admin.ModelAdmin):
     readonly_fields = ['created_by']
-    list_display = ['report_date', 'status', 'authority_email']
-    list_editable = ['status', 'authority_email']
+    list_display = ['report_date', 'status', 'email_status', 'email_list']
```

Comment:
Don't exactly know how this works, so I'll assume it does :)

Reply:
Email Status is a boolean field in the model so it shoes up properly. Email List is a callable i.e a function in the Complaints model which returns a list of the Authorities associated with the complaint